### PR TITLE
Medications crud

### PR DESCRIPTION
**Pull request by kirkkwang**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import PatientInfo from "./pages/PatientInfo";
 import PatientNew from "./pages/PatientNew";
 import PatientEdit from "./pages/PatientEdit";
 import "./App.css";
+import { resolveConfig } from "prettier";
```
Comment (archae0pteryx) [inline app/javascript/components/App.js:14]: I've never seen this before. what do?

Comment (kirkkwang) [reply app/javascript/components/App.js:14]: oh hmm, i didn't catch that, i don't know if it actually does anything, i'll investigate

File: app/javascript/components/pages/MedicationEdit.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+export default class MedicationEdit extends Component {
+  constructor(props) {
+    super(props);
+    const { medication_name, dose, frequency, time, route, tx, prescribed_by } =
+      this.props.medication;
+
+    this.state = {
+      updateMedication: {
+        medication_name: medication_name ? medication_name : "",
```
Comment (archae0pteryx) [inline app/javascript/components/pages/MedicationEdit.js:13]: another, fancy wat to write this is `medication_name ?? ""`

File: app/javascript/components/pages/MedicationNew.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+  handleSubmit = () => {
+    this.props.createMedication(this.state.newMedication);
+    this.setState({ submitted: true });
+  };
+
+  render() {
+    const { medication_name, dose, frequency, time, prescribed_by, tx, route } =
+      this.state.newMedication;
+    console.log(this.state.newMedication);
```
Comment (archae0pteryx) [inline app/javascript/components/pages/MedicationNew.js:36]: console.log? more like console.nope!

File: app/javascript/components/pages/MedicationShow.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+
+export default class MedicationShow extends Component {
+  render() {
+    const { medication_name, dose, frequency, time, route, tx, prescribed_by } =
+      this.props.medication;
+    return (
+      <>
+        {/* <h3>Medication Show</h3>
```
Comment (archae0pteryx) [inline app/javascript/components/pages/MedicationShow.js:9]: you shouldn't commit comments. If you need it later make a commit so you can go back to get it.

File: app/javascript/components/pages/PatientEdit.js [hunk 5]
```diff
@@ -1,12 +1,12 @@
 import React, { Component } from "react";
-import { Redirect } from "react-router-dom";
+import { Redirect, Link } from "react-router-dom";
 import { Form, FormGroup, Input, Label, Button } from "reactstrap";
 // import "./PatientNew.css";
```
Comment (archae0pteryx) [inline app/javascript/components/pages/PatientEdit.js:4]: no comment.

**Review by archae0pteryx — APPROVED** [review]
borat voice: very nice. just remove the comments and log and you're gold IMO

File: app/javascript/components/pages/MedicationNew.js (hunk 3 above)
Comment (kirkkwang) [reply app/javascript/components/pages/MedicationNew.js:36]: dang it, yes i'll fix!

File: app/javascript/components/pages/MedicationShow.js (hunk 4 above)
Comment (kirkkwang) [reply app/javascript/components/pages/MedicationShow.js:9]: got it, will remove

**Pull request merged**